packages/admin-stories/src/docs/comet/admin/savesplitbutton.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+import { SaveSplitButton, SaveButton } from "@comet/admin";
+import { FormattedMessage } from "react-intl";
+
+# Components
+
+<Meta title="Docs/@comet/admin/savesplitbutton" />
+
+## Save / Save and go Back Split Button
+
+<Canvas>
+    <Story name="Save / Save and go back">
+        {() => {
+            const Story = () => {
+                const [saving, setSaving] = React.useState(false);
+                return (
+                    <SaveSplitButton localStorageKey={"Page1.SaveSplitButton"} saving={saving} variant={"contained"} color={"primary"}>
+                        <SaveButton
+                            saving={saving}
```

Comment:
It would be nice if `saving` was passed through from `SaveSplitButton`. You could use two different approaches to achieve this:
1. Use a Context
2. Clone the `children` of `SaveSplitButton` and pass through `saving`. (you might need two different props interfaces for the children for optimal type safety

Reply:
we discussed already discussed that. i removed the wrong, confusing and duplicated saving prop. That should only be on the SaveButton